config/crds/tenancy.kcp.dev_clusterworkspaces.yaml

```diff
@@ -221,7 +221,7 @@ spec:
                   description: ClusterWorkspaceInitializer is a unique string corresponding
                     to a cluster workspace initialization controller for the given
                     type of workspaces.
-                  pattern: ^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?)$
+                  pattern: ^(root|kcp-dev)(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?)$
```

Comment:
Not quite what I had in mind - was looking to match either:
- `root` / `root:...`
- `kcp.dev/<anything>`